examples/wind/src/layers/particle-layer/particle-layer.js

```diff
@@ -332,24 +334,21 @@ export default class ParticleLayer extends Layer {
   }
 
   createTexture(gl, opt) {
```

Comment:
Yeah, if you look at the flow, they are never passed down to Texture2D constructor.